constants/projects.json

```
@@ -1,16 +1,20 @@
 {
     "data": [
         {
-            "img": "https://example.com",
             "title": "Morrent",
+            "mobileImg": "@/public/project-images/Morrent/Morrent-Mobile.png",
+            "laptopImg": "/public/project-images/Morrent/Morrent-Laptop.png",
```

```suggestion
            "laptopImg": "/project-images/Morrent/Morrent-Laptop.png",
```

"public" is the folder that gets exposed, so we shouldn't include it :)